README.md

```diff
@@ -171,15 +211,17 @@ To print out debug logs:
 python -m autogpt --debug
 ```
 
-### Docker
+### <a name="docker"></a>Docker
 
 You can also build this into a docker image and run it:
 
 ```bash
 docker build -t autogpt .
 docker run -it --env-file=./.env -v $PWD/auto_gpt_workspace:/app/auto_gpt_workspace autogpt
 ```
-
+#### Installing Docker
+- To install Docker on your machine please follow the [Official Docker Engine Installation Guide](https://docs.docker.com/engine/install/)
+  
```

Comment:
Please move to below `docker-compose` instructions.

Reply:
Moved accordingly